src/main/java/Task.java

```diff
@@ -0,0 +1,34 @@
+public class Task {
+
+    protected String taskName;
+    protected boolean isDone;
+    public Task(String taskName){
+        this.taskName = taskName;
+        this.isDone = false;
+    }
+
+    public String getTaskName() {
+        return taskName;
+    }
+
+    public void setTaskName(String taskName) {
+        this.taskName = taskName;
+    }
+
+    public boolean isDone() {
+        return isDone;
+    }
+
+    public void setDone(boolean done) {
```

Comment:
I think the parameter name should be `isDone`, to be consistent with the member's name.

Reply:
Thanks @danbaterisna